lib/dynamoid/fields.rb

```diff
@@ -294,6 +294,7 @@ def generated_methods
     # @since 0.2.0
     def write_attribute(name, value)
       name = name.to_sym
+      old_value = __send__(name)
```

Comment:
`read_attribute` looks better than `__send__`

Reply:
Thanks, I fixed it.